pontoon/static/css/dark-theme.css

```diff
@@ -0,0 +1,68 @@
+/* Dark Theme Variables */
+:root {
+  --black-1: #1c1e21;
+  --black-2: #000000;
+  --black-3: #000;
```

Comment:
Note that to convert a 3-value hex syntax to a 6-value syntax, each value needs to be duplicated:
https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color#syntax

So:
* `#000` is the same as `#000000`
* `#FFF` is the same as `#FFFFFF`
* `#123` is the same as `#112233`
* etc.

I suggest we use the six-value (or eight-value when _alpha_ is requried) hex syntax everywhere.